cmd/orderimports/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - dims
+  - cblecker
+  - spiffxp
+approvers:
+  - dims
+  - cblecker
+  - spiffxp
```

Comment:
Not sure if this is the desired list of owners. I would assume who ever wrote this to own the code too.

Reply:
Most of the code refers to the implementation of  [preferredimports](https://github.com/kubernetes/kubernetes/blob/master/cmd/preferredimports/preferredimports.go), [preferredimports 's OWNER](https://github.com/kubernetes/kubernetes/blob/master/cmd/preferredimports/OWNERS)  

I'm not sure if this is appropriate. I apply to join the kubernetes org first, and then add myself to the owner list?

Reply:
you could add yourself later.

> I apply to join the kubernetes org first, and then add myself to the owner list?

do you have sufficient contributions?

Reply:
Not sure if there is sufficient contribution. I created these [PRs](https://github.com/pulls?q=is%3Apr+author%3Axinydev+org%3Akubernetes+org%3Akubernetes-sigs)